tests/fixed/type-hints.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TypeHints;
+
+use Iterator;
+use Traversable;
+
+class TraversableTypeHints
+{
+    /** @var Traversable */
+    private $parameter;
+
+    /**
+     * @param Iterator $iterator
+     *
+     * @return Traversable
```

Comment:
How is this type hint still in the fixed file? Shouldn't this have a specification, like `Traversable<string>`?

Reply:
Yes but it can't be automatically fixed, AFAIK that's what fixed folder is for, hm? So after running `phpcbf` it does not change

Reply:
Ok, I understand. At first I thought it should remove it and then I realised that it shouldn't because it needs a generic type hint. Just didn't make the final connection, thanks for that!